resources/views/user/delete.twig

```diff
@@ -0,0 +1,18 @@
+{% extends "layouts/default.twig" %}
+
+{% block scripts %}
+    {{ parent() }}
+{% endblock %}
+
+{% block content %}
+    <div class="text-center">
+            <h2 class="headline">Delete My Profile</h2>
+            <p>
+                By clicking on the button below all data associated with your account<br>
```

Comment:
I feel that the usage of `<br>` tag here is for cosmetic reasons - maybe we would be better of by using a container for the entire block that limits the width of what is contained?

Reply:
I will need some help from someone with better layout skills to do this for me right now